src/shared/styles-constants.ts

```diff
@@ -0,0 +1,3 @@
+export const FULL_PAGE_MAX_WIDTH = 440;
+export const POPUP_WIDTH = 442;
+export const POPUP_HEIGHT = 646;
```

Comment:
Are these used outside `src/app`? If not, they should go in there.

Reply:
Two of them are used in `src/background`.

Reply:
I'll move those back into `constants.ts` and the other into `src/app`